lua/hbac/command/subcommands.lua

```diff
@@ -28,14 +28,8 @@ M.toggle_autoclose = function()
 	vim.notify("Autoclose " .. autoclose_state, vim.log.levels.INFO, notify_opts)
 end
 
-M.telescope = function(opts)
-	local hbac_telescope = require("hbac.telescope")
-	if not hbac_telescope then
-		return
-	end
-	local telescope_opts = require("hbac.config").values.telescope
-	opts = vim.tbl_deep_extend("force", telescope_opts, opts or {})
-	hbac_telescope.pin_picker(opts)
+M.telescope = function()
+	vim.notify('This command has been removed. Check the docs for a migration guide.', vim.log.levels.WARN)
```

Comment:
Are you both ok with this? I didn't want to include the new command in the message, as it won't work if you don't explicitly load the extension in your telescope config.

Reply:
Looks good! and good point! Thanks

Reply:
I just realized that i forgot to add the notify_opts, so I'll fix that real quick.